scripts/validation_test.sh

```diff
@@ -137,9 +137,9 @@ EOF
 watch_validation() {
     log::info "watch validation test running..."
 
-    command="${ETCDCTL} watch watch_key"
+    command="${ETCDCTL} watch watch_key -- echo watch event received"
```

Comment:
Is it better to add a new test rather than modify the existing one? like `watch_with_command_validation` This means that the original watch test still needs to be retained.